types/testdouble/genesis.go

```diff
@@ -0,0 +1,25 @@
+package testdouble
+
+import (
+	"github.com/tendermint/tendermint/crypto/ed25519"
+	cmn "github.com/tendermint/tendermint/libs/common"
+	"github.com/tendermint/tendermint/types"
+	tmtime "github.com/tendermint/tendermint/types/time"
+)
+
+// RandomGenesisDoc returns a random genesis document with one validator.
+// Genesis time is set to now().
+func RandomGenesisDoc() *types.GenesisDoc {
```

Comment:
do folks think this is a good idea? maybe we should just export the same method from `types` package?

Reply:
I didn't totally understand why this happened.

Reply:
Wanted to reuse RandomGenesisDoc() function.